src/app/beer_garden/api/http/base_handler.py

```diff
@@ -70,7 +69,7 @@ class BaseHandler(RequestHandler):
         NotUniqueException: {"status_code": 409},
         NotUniqueError: {"status_code": 409, "message": "Resource already exists"},
         EndpointRemovedException: {"status_code": 410, "message": "Endpoint removed"},
-        DocumentTooLarge: {"status_code": 413, "message": "Resource too large"},
```

Comment:
I might just remove this one to be honest.  I don't think `InvalidDocumentError` is a drop in replacement here and neither of these is actually what gets raised in the scenario where you call `save()` on an overly large document.